modules/ROOT/pages/manage-backup-restore.adoc

```diff
@@ -13,6 +13,11 @@ To preserve applications in the event of system failure, ensure that Runtime Fab
 MuleSoft highly recommends configuring hourly backups. Store your backup archive on external storage outside 
 the Runtime Fabric cluster.
 
+[NOTE]
+The state of Runtime Fabric is distributed across its controller nodes. Use the following procedures to backup and restore
```

Comment:
Not sure what this sentence means: The state of Runtime Fabric is distributed across its controller nodes.